### ISSUE TITLE
Add stacked row charts

### ISSUE THREAD

**Issue by jefffriesen** [dc-js/dc.js#397]
Nick said that doing this would probably be more than a renderlet could do. 

I would love this feature. Especially if you could choose between a regular row chart like this:
![screen shot 2013-11-05 at 2 19 41 pm](https://f.cloud.github.com/assets/445733/1477689/035503c8-4660-11e3-9b92-3cc74449bbad.png)

Or normalized (100%) row chart like this:
http://dimplejs.org/examples_viewer.html?id=bars_horizontal_stacked_100pct
![screen shot 2013-11-05 at 2 22 44 pm](https://f.cloud.github.com/assets/445733/1477717/7a37e30c-4660-11e3-9949-71372fe1bf98.png)

For reference, here is a normalized vertical bar chart from Mike Bostock:
http://bl.ocks.org/mbostock/3886394
![screen shot 2013-11-05 at 2 24 39 pm](https://f.cloud.github.com/assets/445733/1477749/b5fe5948-4660-11e3-93b2-5ad6bb184fa8.png)

**Comment by carlthuringer** [comment]
@RohanHart :+1: Just integrated [your fork](https://github.com/RohanHart/dc.js) in a project that called for stacked row chart. Works flawlessly, compatible with the other charts we were building.

I would really like to see this integrated. Thanks for the hard work.

**Comment by RohanHart** [comment]
Glad it worked for for you.  Hopefully being a year behind the master wasn't too painful.

**Comment by flxfxp** [comment]
Any news on this? I'd love to have stacked row bars :)

**Comment by jennakwon06** [comment]
Do you have an example or demo for the stacked row chart?

**Comment by rusosnith** [comment]
bump! what happened to this?

**Comment by gordonwoodhull** [comment]
Hi @rusosnith. It would be great to have this feature, but no one has contributed it. If you want to help, you could take a look at @RohanHart's fork and see how hard it would be to bring this feature into the dc.js library. Or I think there have been other attempts, but none resulted in a PR.

**Comment by renauld94** [comment]
Hi @gordonwoodhull  is it still in developement?

**Comment by gordonwoodhull** [comment]
Hi @renauld94. When I looked at @RohanHart's fork way back when, I wasn't able to piece apart the changes for this feature versus other changes he made. It has been a while but if anyone wants to figure this out, the contribution will be welcome!

**Comment by rusosnith** [comment]
Damn. I got here again. To ask what happened to this. And then realize I had already asked the same thing 2 years ago

**Comment by gordonwoodhull** [comment]
Yep, no one has contributed this yet. Glad to help you out if you want to give it a try!

**Comment by gordonwoodhull** [comment]
FWIW, the normalization part of the original issue is easy to solve with a crossfilter custom reduction, see [this Stack Overflow question](https://stackoverflow.com/questions/61710263/display-percentage-of-dimension-over-sum-of-data-using-dc-rowchart/61739545#61739545). 

This issue is still about stacked rows, which still don't exist, but I wanted to point out that normalization is bread and butter for crossfilter, even nicer with [reductio](https://github.com/crossfilter/reductio), and doesn't need to be implemented in dc.js.
